test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/NuGet.XPlat.FuncTest.csproj

```diff
@@ -9,15 +9,15 @@
     <TestProjectType>functional</TestProjectType>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Microsoft.Build.Runtime" Version="15.3.409" />
-    <PackageReference Include="Microsoft.CodeAnalysis.Build.Tasks" Version="3.0.0-beta1-61516-01" />
-    <PackageReference Include="Microsoft.CodeAnalysis.CSharp" Version="3.0.0-beta1-61516-01" />
-    <PackageReference Include="System.Collections.Immutable" Version="1.3.1" />
+    <PackageReference Include="Microsoft.Build.Runtime" Version="16.0.0-preview.256" />
+    <PackageReference Include="Microsoft.CodeAnalysis.Build.Tasks" Version="3.0.0-dev-61717-03" />
```

Comment:
Why these specific versions?
Where are these packages coming from?

Reply:
It's test so I guess not so important :)

Reply:
They where just the latest